src/type/introspection.js

```diff
@@ -16,7 +16,7 @@ import {
   GraphQLEnumType,
   GraphQLInputObjectType,
   GraphQLList,
-  GraphQLNonNull
+  GraphQLNonNull,
```

Comment:
We like trailing commas to avoid unattributed changes to lines later, but haven't been picky about them.

I just thought it was hilarious that the only change to this file was to add a comma.